### PR TITLE
Change color of text and delete button border

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

**Comment by cehali** [comment]
Before: 
image
After:
image

File: universal-login-react/src/ui/styles/emoji.sass [hunk 1]
```diff
@@ -313,14 +313,15 @@
     flex-direction: column
 
   .emoji-success-text
```
Comment (xitronix) [inline universal-login-react/src/ui/styles/emoji.sass]: This style will affect text color in jarvis. Universal Login styles should be in default.

**Comment by xitronix** [comment]
image

File: universal-login-react/src/ui/styles/emoji.sass (hunk 1 above)
Comment (cehali) [reply universal-login-react/src/ui/styles/emoji.sass]: Fixed

**Comment by cehali** [comment]
@xitronix 
image

File: universal-login-react/src/ui/styles/emoji.sass [hunk 2]
```diff
@@ -313,14 +313,15 @@
...
-    color: #000
+    color: #FFFFFF
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/emoji.sass]: move this class to `emojiDefaults.ts`

Comment (cehali) [reply universal-login-react/src/ui/styles/emoji.sass]: Done

File: universal-login-wallet/src/ui/styles/layout/_emoji.sass [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.jarvis-styles
+  .emoji-success-text
+    color: #000
```
Comment (Marik-D) [inline universal-login-wallet/src/ui/styles/layout/_emoji.sass]:
```suggestion
    color: #000
    
```

**Comment by JustynaBroniszewska** [comment]
Ok, let's change the color in the next PR

**Pull request merged**